src/Network/Wai/Handler/Hal.hs

```diff
@@ -152,6 +147,27 @@ defaultOptions =
         _ -> True
     }
 
+-- | A variant of 'run' with configurable 'Options'. Useful if you
+-- just want to override the 'binaryMediaTypes' setting but don't need
+-- the rest of 'runWithContext''s features.
+--
+-- @since 0.4.0.0
+runWithOptions ::
+  (MonadIO m) =>
+  -- | Configuration options. 'defaultOptions' provides sensible defaults.
+  Options ->
+  Wai.Application ->
+  HalRequest.ProxyRequest HalRequest.NoAuthorizer ->
+  m HalResponse.ProxyResponse
+runWithOptions opts app req =
+  liftIO $ do
```

Comment:
Join to previous line for consistency with the rest of the file

Reply:
Done.